util/retry/retry.go

```diff
@@ -1,20 +1,55 @@
 package retry
 
 import (
+	"os"
+	"strconv"
 	"time"
 
+	log "github.com/sirupsen/logrus"
+
 	"k8s.io/apimachinery/pkg/util/wait"
+
+	envutil "github.com/argoproj/argo/v2/util/env"
 )
 
-// DefaultRetry is a default retry backoff settings when retrying API calls
-// Retry   Seconds
-//     1      0.01
-//     2      0.03
-//     3      0.07
-//     4      0.15
-//     5      0.31
-var DefaultRetry = wait.Backoff{
-	Steps:    5,
-	Duration: 10 * time.Millisecond,
-	Factor:   2,
+// BackoffSettings obtains the retry backoff settings when retrying API calls.
+func BackoffSettings() wait.Backoff {
```

Comment:
I think this can be unchanged  just be a var called `DefaultRetry`, use an `init()` func to set its values

Reply:
Thanks. Updated.